src/rp2_common/pico_stdio_usb/reset_interface.c

```diff
@@ -6,6 +6,7 @@
 #include "tusb.h"
 
 #include "pico/bootrom.h"
+#include "pico/platform.h"
```

Comment:
we don't generally include `pico/platform.h` directly unless this is a non SDK tool (usually `pico.h` is included because it also includes all user/board cinfig), and that should be included by all other library headers (as it is by `pico/bootrom.h`) so, i'd rather not include this